content/vendor/cli/installer create.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2021-09-10
+linktitle: "installer create"
+title: "installer create"
+weight: 90115
+---
+
+Create a new installer spec by providing YAML configuration for a https://kurl.sh cluster.
+
+### Usage
+```bash
+replicated installer create [flags]
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+|  `--auto`            | |    generate default values for use in CI
+| `-y, --confirm-auto` | |       auto-accept the configuration generated by the --auto flag
+|  `--ensure-channel`   | |  When used with --promote <channel>, will create the channel if it doesn't exist
```

Comment:
When used with --promote <channel>, this command creates the channel if it doesn't already exist